itensor/iterativesolvers.h

```diff
@@ -1163,10 +1174,10 @@ applyExp(BigMatrixT const& A,
 
         // Update w = beta * V * exp(sgnt * t_step * H) * e1
         mx = mbrkdwn + std::max(0,k1-1);
-        w = V[0]*F(0);
+        w = (V[0]*=F(0,0));
         for(int i = 1; i < mx; ++i)
             {
-            w += V[i]*F(i);
+            w += (V[i]*=F(i,0));
```

Comment:
Can this just be `w += V[i]*F(i,0)`? Also a similar question for the line right before the loop.

Reply:
I thought *= will avoid creating a temporary ITensor?

Reply:
Hmm yes I suppose here it might. Though maybe it is better written as
```C++
V[i] *= F(i,0);
w += V[i];
```
just for clarity.

But I think the change before the loop is not necessary.